keps/sig-network/2091-admin-network-policy/README.md

```diff
@@ -402,129 +404,124 @@ type AdminNetworkPolicy struct {
   Egress      []AdminNetworkPolicyEgressRule
 }
 
-// AdminNetworkPolicySubject defines what objects the policy selects. 
-// Exactly one of the selector pointers should be set.  
+// AdminNetworkPolicySubject defines what objects the policy selects.
+// Exactly one of the `Namespaces` or `NamespacedPods` pointers should be set.
 type AdminNetworkPolicySubject struct {
-  NamespaceSelector         *metav1.LabelSelector
-  NamespaceAndPodSelector   *NamespaceAndPodSelector
+  Namespaces        *metav1.LabelSelector
+  NamespacedPods    *NamespacedPodSubject
 }
 
-// NamespaceAndPodSelector allows the user to select a Namespace(s) and optionally 
-// select a given set of pod(s) in that namespace(s)
-type NamespaceAndPodSelector struct {
-  // This field follows standard label selector semantics; if present but empty, 
-  // it selects all Namespaces.  
-  NamespaceSelector   metav1.LabelSelector
-
-  // Used to explicitly select pods within a namespace; if present but empty, 
-  // it selects all Pods.  
-  PodSelector         metav1.LabelSelector
+// NamespacedPodSubject allows the user to select a given set of pod(s) in
+// selected namespace(s)
+type NamespacedPodSubject struct {
+  NamespaceSelector  *metav1.LabelSelector
+  PodSelector        *metav1.LabelSelector
 }
 
-// AdminNetworkPolicyIngressRule describes an action to take on a particular 
-// set of traffic destined for pods selected by an AdminNetworkPolicy's 
-// Subject field. The traffic must match both l4Selector and from. 
+// AdminNetworkPolicyIngressRule describes an action to take on a particular
+// set of traffic destined for pods selected by an AdminNetworkPolicy's
+// Subject field. The traffic must match both l4Selector and from.
 type AdminNetworkPolicyIngressRule struct {
   // Name is an identifier for this rule.
   // +optional
   Name         string
 
-  // Action specifies whether this rule must pass, allow or deny traffic. 
-  // Allow: allows the selected traffic 
+  // Action specifies whether this rule must pass, allow or deny traffic.
+  // Allow: allows the selected traffic
   // Deny: denies the selected traffic
-  // Pass: allows the selected traffic to skip and remaining positive priority ANP rules 
-  // and be delegated by K8's Network Policy. 
+  // Pass: allows the selected traffic to skip and remaining positive priority ANP rules
+  // and be delegated by K8's Network Policy.
   Action       AdminNetPolRuleAction
-  
-  // L4Selector allows for matching traffic based on L4 constructs. 
-  L4Selector   AdminNetworkPolicyL4Selector
 
-  // List of sources from which traffic will be allowed/denied/passed to the entities 
-  // selected by this AdminNetworkPolicyRule. Items in this list are combined using a logical OR 
-  // operation. If this field is empty, this rule matches no sources. 
+  // L4Selector allows for matching traffic based on L4 constructs.
+  L4Selector   AdminNetworkPolicyPorts
```

Comment:
It sounds like they don't like this naming :)

Reply:
Done. Using Tim's second suggestion for now